packages/components-react/paragraph/src/paragraph.test.tsx

```diff
@@ -0,0 +1,13 @@
+import '@testing-library/jest-dom';
+import { render } from '@testing-library/react';
+import { Paragraph } from './paragraph';
+
+describe('paragraph', () => {
```

Comment:
Misschien toch `Paragraph` gebruiken? Voor componenten doen we nu hoofdletters zoals `Radio Group`